diode-server/auth/server.go

```diff
@@ -215,13 +223,7 @@ func (s *Server) getAuthToken(r *http.Request) (string, error) {
 }
 
 func (s *Server) validateToken(jwtToken string) (jwt.MapClaims, error) {
-	jwksURL := s.config.OAuth2.PublicServerURL + "/.well-known/jwks.json"
-	jwks, err := keyfunc.NewDefault([]string{jwksURL})
-	if err != nil {
-		return nil, NewAuthError("failed to get JWKS", http.StatusInternalServerError)
-	}
-
-	token, err := s.tokenParser.Parse(jwtToken, jwks.Keyfunc)
+	token, err := s.tokenParser.Parse(jwtToken, s.keyfunc.Keyfunc)
```

Comment:
If it's preferred to fetch the JWK Set every time, manually perform an HTTP request and use `keyfunc.NewJWKSetJSON`. Here's an example: https://github.com/MicahParks/keyfunc/blob/main/examples/json/main.go